src/Microsoft.Health.Dicom.Functions.UnitTests/Indexing/ReindexDurableFunctionTests.cs

```diff
@@ -7,37 +7,46 @@
 using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
 using Microsoft.Health.Dicom.Core.Features.Indexing;
 using Microsoft.Health.Dicom.Core.Features.Retrieve;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.SqlServer.Features.Schema;
+using Microsoft.Health.SqlServer.Features.Schema.Manager;
 using Microsoft.Health.Dicom.Functions.Indexing;
-using Microsoft.Health.Dicom.Functions.Indexing.Configuration;
 using NSubstitute;
+using Microsoft.Health.Dicom.Operations.Functions.Configs;
 
 namespace Microsoft.Health.Dicom.Functions.UnitTests.Indexing
 {
     public partial class ReindexDurableFunctionTests
     {
-        private readonly ReindexConfiguration _reindexConfig;
+        private readonly ReindexOperationConfiguration _reindexConfig;
         private readonly IReindexStore _reindexStore;
         private readonly IInstanceReindexer _instanceReindexer;
         private readonly IAddExtendedQueryTagService _addExtendedQueryTagService;
         private readonly IInstanceStore _instanceStore;
         private readonly IExtendedQueryTagStore _extendedQueryTagStore;
         private readonly ReindexDurableFunction _reindexDurableFunction;
+        private readonly ISchemaManagerDataStore _schemaManagerDataStore;
+        private readonly SchemaInformation _schemaInformation;
 
         public ReindexDurableFunctionTests()
         {
-            _reindexConfig = new ReindexConfiguration();
+            _reindexConfig = new ReindexOperationConfiguration();
             _reindexStore = Substitute.For<IReindexStore>();
             _instanceReindexer = Substitute.For<IInstanceReindexer>();
             _addExtendedQueryTagService = Substitute.For<IAddExtendedQueryTagService>();
             _instanceStore = Substitute.For<IInstanceStore>();
             _extendedQueryTagStore = Substitute.For<IExtendedQueryTagStore>();
+            _schemaManagerDataStore = Substitute.For<ISchemaManagerDataStore>();
+            _schemaInformation = new SchemaInformation(SchemaVersionConstants.Min, SchemaVersionConstants.Max);
             _reindexDurableFunction = new ReindexDurableFunction(
-                Options.Create(new IndexingConfiguration() { Add = _reindexConfig }),
+                Options.Create(new DicomFunctionsConfiguration() { Reindex = _reindexConfig }),
```

Comment:
Any particular reason we're not just Substituting for `IOptions<DicomFunctionsConfiguration>`?

Reply:
Also, wouldn't it be `DicomFunctionsConfiguration.ServicesConfiguration.Reindex` or am I missing something??

Reply:
* just moved to Substitution
* I'm not sure if we want to add another layer (`Services`) because so far we don't have other things like `Security`, `Features` etc.